lib/qualys/element.rb

```diff
@@ -99,15 +99,20 @@ def cleanup_html(source)
       result.gsub!(/<p>/i, "\n\n")
       result.gsub!(/<br>/i, "\n")
       result.gsub!(/          /, "")
-      result.gsub!(/<a href=\"(.*?)\" target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
+      result.gsub!(/<a href=\"(.*?)\"\s?target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
```

Comment:
why is this change needed?

Reply:
Several instances in the sample file don't have the space before `target` (e.g. `<a href=""target="">`) and weren't being caught and replaced. This resolves the issue completely.

Reply:
but that's an error on their side, not ours!

Reply:
I'd rather not have our users suffer when it's a simple regex update! 😄